### PR TITLE
Add OnFinality

### PR DESCRIPTION
**Pull request by jamesbayly**

### CODE REVIEW

**Comment by arddluma** [comment]
Hello @jamesbayly,
Thanks for your contribution, 
Could you please remove the text in multichain section because we are no longer accepting entries there!

File: README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ most of the Networks such as Ethereum, BSC, Polygon, Gnosis, Fantom, Avalanche,
   - [Pocket Network](#pocket-network)
   - [Quicknode](#quicknode)
   - [Chainstack](#chainstack)
+  - [OnFinality](#onfinality)
```
Comment (arddluma) [inline README.md:20]: Remove this line

File: README.md [hunk 2]
```diff
@@ -105,6 +107,14 @@ The nodes are deployed in multiple regions around the globe using multi-cloud, h
 
 Checkout: [chainstack.com](https://chainstack.com/)
 
+### OnFinality
```
Comment (arddluma) [inline README.md:110]: Please remove this

**Comment by jamesbayly** [comment]
> Hello @jamesbayly, Thanks for your contribution, Could you please remove the text in multichain section because we are no longer accepting entries there!

Would there be a minimum number of chains that we would support in order to get on that list?

File: README.md [hunk 3]
```diff
@@ -487,6 +487,12 @@ Checkout: [chainstack.com](https://chainstack.com/)
 
 - [https://near-nodes.io/rpc/hardware-rpc#recommended-hardware-specifications](https://near-nodes.io/rpc/hardware-rpc#recommended-hardware-specifications)
 
+## Cosmos
```
Comment (arddluma) [inline README.md:490]: Could you please add public RPC for Cosmos if you have any?

**Pull request merged**